src/qml/components/BlockClock.qml

```diff
@@ -42,7 +42,7 @@ Item {
         id: dial
         anchors.horizontalCenter: root.horizontalCenter
         scale: Theme.blockclocksize
-        width: Math.min((root.parentWidth * dial.scale), (root.parentHeight * dial.scale))
+        width: Math.max(Math.min(200, Math.min(root.parentWidth - 30, root.parentHeight - 30)), Math.min((root.parentWidth * dial.scale), (root.parentHeight * dial.scale)))
```

Comment:
Can we break this up into two lines? It's difficult to read.

Reply:
Yes on both counts... new commit with changes https://github.com/bitcoin-core/gui-qml/pull/385/commits/d6331aa60f46ceb8d6d5c2019b7d64d6ec16310a